src/Bundle/Resources/config/services.xml

```diff
@@ -69,7 +69,7 @@
         <service id="Sylius\Bundle\ResourceBundle\Form\Type\DefaultResourceType" alias="sylius.form.type.default" />
 
         <service id="sylius.registry.resource_repository" class="Sylius\Component\Registry\ServiceRegistry" public="false">
-            <argument>Sylius\Component\Resource\Repository\RepositoryInterface</argument>
+            <argument>Doctrine\Persistence\ObjectRepository</argument>
```

Comment:
This feels like a bigger change, is there some explanation why `sylius/resource` goes away from `RepositoryInterface`?

What will happen, if my custom repository doesn't implement any resource trait and interface?

Reply:
yes, the reason is that we do not need to implement that specific interface anymore in the new routing system.
This is the reason why we have this issue (cf first error message on first screenshot).

RepositoryInterface from Sylius extends that ObjectRepository from Doctrine.